Apps/phgitlab/gitlab.json

```diff
@@ -0,0 +1,439 @@
+                },
+                {
+                    "data_path": "action_result.data.*.*.name",
+                    "data_type": "string",
+                    "example_values": [
+                        "project-name"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.*.path",
+                    "data_type": "string",
+                    "example_values": [
+                        "path-to/project-name"
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "list branches",
+            "identifier": "list_branches",
+            "description": "List the branches of a GitLab project",
+            "verbose": "List the branches of a GitLab project",
+            "type": "generic",
+            "read_only": true,
+            "parameters": {
+                "project_id": {
+                    "description": "The ID of the GitLab project",
+                    "data_type": "numeric",
+                    "required": true,
+                    "primary": true,
+                    "contains": [],
+                    "value_list": [],
+                    "default": "",
+                    "order": 0
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.project_id",
+                    "data_type": "numeric",
+                    "contains": [],
+                    "column_name": "project_id",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "summary.total_objects",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "summary.total_objects_successful",
+                    "data_type": "numeric"
+                },
+                {
+                    "data_path": "action_result.data.*.*.name",
+                    "data_type": "string",
+                    "example_values": [
+                        "master"
+                    ]
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "create trigger",
+            "identifier": "create_trigger",
+            "description": "Create a new pipeline trigger for a GitLab project",
+            "verbose": "Create a new pipeline trigger for a GitLab project",
```

Comment:
`description` and `verbose` are the same, please follow as mentioned above

Reply:
I deleted the `verbose` in [this commit](https://github.com/peterbertel/phantom-apps/commit/8159a235f9d415edc12c6442f09df319460868e2).